src/main/webapp/scripts/view.js

```diff
@@ -0,0 +1,79 @@
+/** Botton that triggers authorization/sign out flow */
```

Comment:
Whole PR: You added a controller and view logic. You need to add tests to make sure it works :). Mock the responses from the API and test that your controller does what it does and sets the view correctly. Mocking the listen probably won't work since its probably doing some fancy stuff in it. So you probably are fine with just calling the methods directly and making sure the view is changed.

Reply:
Ok that makes sense. Just to confirm are you referring to some kind of integration test involving DOM instead of unit testing a single function like handleCommentClick()? (your comment on Jennifer's PR about quit-dom will be helpful :D)

Reply:
wow it's cool that github notified me that I was mentioned!
I asked a question about this before and learned that testing controller functions (eg. get some data, do an api call, get response, display response) is considered a unit test and testing the helper methods is not always necessary (see comment on PR #12)